Automattic/wp-calypso
# A4A: Add Billing summary header.

This pull request adds the A4A billing summary header.

<img width="1557" alt="Screenshot 2024-03-01 at 5 35 14 PM" src="https://github.com/Automattic/wp-calypso/assets/56598660/3d31b9ff-f920-4e5b-85d6-fa8fbe062020">

Closes https://github.com/Automattic/jetpack-genesis/issues/269
Closes https://github.com/Automattic/jetpack-genesis/issues/267

## Proposed Changes

* Add mock data for the Billing dashboard.
* Port the Summary header from the Jetpack Manage Billing page.


## Testing Instructions
* Switch branch: `git checkout add/a4a-billing-summary-header`.
* Start the server by running `yarn start-a8c-for-agencies`.
* Go to http://agencies.localhost:3000/purchases/billing.
* Confirm that the Billing summary header is visible.

## Pre-merge Checklist
